waspc/src/Wasp/Generator/ServerGenerator/EmailSenderG.hs

```diff
@@ -94,7 +100,7 @@ depsRequiredByEmail spec = maybeToList maybeNpmDepedency
   where
     maybeProvider :: Maybe Providers.EmailSenderProvider
     maybeProvider = getEmailSenderProvider <$> (AS.App.emailSender . snd . getApp $ spec)
-    maybeNpmDepedency = Providers.npmDependency <$> maybeProvider
+    maybeNpmDepedency = maybeProvider >>= Providers.npmDependency
```

Comment:
This is not just a simple rewrite, something changed types here, right?

Reply:
The `npmDepedency` turned into a `Maybe` since the `Dummy` sender doesn't have any deps to install alongside it.